public/pages/Findings/components/FindingDetailsFlyout.tsx

```diff
@@ -217,12 +225,16 @@ export default class FindingDetailsFlyout extends Component<
           </EuiFlexItem>
           <EuiFlexItem grow={false}>
             <EuiButton
-              href={
-                indexPatternId
-                  ? `discover#/context/${indexPatternId}/${related_doc_ids[0]}`
-                  : `#${ROUTES.FINDINGS}`
-              }
-              target={indexPatternId ? '_blank' : undefined}
+              onClick={() => {
+                if (indexPatternId) {
+                  window.open(
```

Comment:
we should instead use the history object for consistency

Reply:
https://github.com/opensearch-project/security-analytics-dashboards-plugin/pull/366#discussion_r1083442982